charts/nextcloud/templates/deployment.yaml

```diff
@@ -238,12 +255,17 @@ spec:
         - name: nextcloud-main
           mountPath: /var/www/html/themes
           subPath: {{ ternary "themes" (printf "%s/%s" .Values.nextcloud.persistence.subPath "themes") (empty .Values.nextcloud.persistence.subPath) }}
+        {{- if .Values.metrics.nginx.enabled }}
+        - name: nginx-metrics-config
+          mountPath: /etc/nginx/conf.d/metrics.conf
+          readOnly: true
+          subPath: metrics.conf
+        {{- end }}
         - name: nextcloud-nginx-config
           mountPath: /etc/nginx/nginx.conf
           subPath: nginx.conf
-        {{- if .Values.nextcloud.extraVolumeMounts }}
```

Comment:
Why is this removed and nextcloud.volumeMounts added? It adds a bunch of volume mounts that are not needed for the nginx container, so why would you include them?

Reply:
I had the fix for https://github.com/nextcloud/helm/pull/358 in this PR first, but then that one got merged, and then I rebased. Now I think something may have gone wrong during rebase.
It was my intention not to have an if statement, because that's how it's specified for the Nextcloud container as well. But I should have used `extraVolumeMounts` instead of `volumeMounts`.

Would you prefer I revert this, of change my version to `extraVolumeMounts`?

Reply:
Better to revert it and try to not get unrelated changes in the PR.

Reply:
Done